airbyte-integrations/connectors/destination-deepset/metadata.yaml

```
@@ -0,0 +1,40 @@
+metadataSpecVersion: "1.0"
+data:
+  name: Deepset
+  icon: deepset.svg
+  definitionId: a6fe9a28-7377-4d2d-aa39-15bcf9578e17
+  connectorBuildOptions:
+    # Please update to the latest version of the connector base image.
+    # Please use the full address with sha256 hash to guarantee build reproducibility.
+    # https://hub.docker.com/r/airbyte/python-connector-base
+    baseImage: docker.io/airbyte/python-connector-base:3.0.0@sha256:1a0845ff2b30eafa793c6eee4e8f4283c2e52e1bbd44eed6cb9e9abd5d34d844
+  connectorTestSuitesOptions:
+    - suite: unitTests
+    - suite: integrationTests
+    - suite: acceptanceTests
+  connectorType: destination
+  dockerRepository: airbyte/destination-deepset
+  dockerImageTag: 0.1.0
+  license: MIT
+  documentationUrl: https://docs.airbyte.com/integrations/destinations/deepset
+  githubIssueLabel: destination-deepset
+  releaseDate: 2025-01-05
+  connectorSubtype: api
+  releaseStage: alpha
+  supportLevel: community
+  tags:
+    - language:python
+    - cdk:python
+    - keyword:ai
+  registryOverrides:
+    oss:
+      enabled: true
+    cloud:
+      enabled: true
+  allowedHosts:
+    hosts:
+      - "*.us.deepset.ai"
+      - "*.cloud.deepset.ai"
+  ab_internal:
+    sl: 200
```

```suggestion
    sl: 100
    ql: 100
```

These are internal to signal the support level we're going for. We usually start with a 100.